app/src/main/java/com/example/cupcake/ui/StartOrderScreen.kt

```diff
@@ -83,7 +84,7 @@ fun StartOrderScreen(
             quantityOptions.forEach { item ->
                 SelectQuantityButton(
                     labelResourceId = item.first,
-                    onClick = {}
+                    onClick = {onNextButtonClicked(item.second)}
                 )
```

Comment:
Please break item into named vals
```suggestion
quantityOptions.forEach { (stringRes, optionValue) ->
SelectQuantityButton(
labelResourceId = item.first,
                    onClick = {onNextButtonClicked(item.second)}
)
```